StripePaymentSheet/StripePaymentSheetTests/PaymentSheet/PaymentSheetLPMConfirmFlowTests.swift

```diff
@@ -108,19 +111,25 @@ final class PaymentSheet_LPM_ConfirmFlowTests: XCTestCase {
 
     func testGrabPayConfirmFlows() async throws {
         // GrabPay has no input fields
-        try await _testConfirm(intentKinds: [.paymentIntent],
+        try await _testConfirm(intentKinds: [.paymentIntent, .deferredPaymentIntent],
                                currency: "SGD",
                                paymentMethodType: .grabPay,
                                merchantCountry: .SG) { _ in
 
         }
     }
+    func testBLIKConfirmFlows() async throws {
+        try await _testConfirm(intentKinds: [.paymentIntent], currency: "PLN", paymentMethodType: .dynamic("blik"), merchantCountry: .BE) { form in
+            form.getTextFieldElement("BLIK code")?.setText("123456")
+        }
+    }
 }
 
 // MARK: - Helper methods
 extension PaymentSheet_LPM_ConfirmFlowTests {
     enum IntentKind: CaseIterable {
         case paymentIntent
+        case deferredPaymentIntent
```

Comment:
AFAIU Blik is only unsupported for deferred *server-side confirmation* but is supported for deferred *client-side confirmation*.

Instead of adding a case here, how about we remove the `deferredSSC` intent just for blik, around line 250?

Reply:
BLIK doesn't show up in the payment sheet anyway when deferredCSC is used.

https://stripe.slack.com/archives/C04HHL5CN1G/p1692125154846659?thread_ts=1690990183.768019&cid=C04HHL5CN1G

<img src="https://github.com/stripe/stripe-ios/assets/137820346/e2499c78-0f58-4aa8-b1f5-622a306b6735" width="50%" />

Reply:
From a technical standpoint I don't see why we can't support Blik in CSC for deferred... but elements/session doesn't even send it back when the intent type is deferred.